tools/cbmc/proofs/HTTP/IotHttpsClient_Connect/Makefile.json

```diff
@@ -1,5 +1,10 @@
 {
   "ENTRY": "IotHttpsClient_Connect",
+
+  # This is the length of the user data after the header in a buffer.
+  # This can be set to any size.
```

Comment:
If this is true, does it make sense to have e.g. three configurations of this proof: with `USER_DATA_SIZE` set to 0, 1, and 1000?

Reply:
Could we leave multiple configurations for after the proofs are completely done?  Unless I'm doing things wrong, they slow down the build-run-debug-commit cycle so much, and we still have so much work to do.

Reply:
Yes, agreed